recipes/asfernandes-cloop/all/idpl.txt

```diff
@@ -0,0 +1,203 @@
+Initial Developer's Public License Version 1.0
```

Comment:
You should not copy a new file directly, instead, extract from source files. Here an example https://docs.conan.io/en/latest/howtos/extract_licenses_from_headers.html

We have other similar cases here in CCI

Reply:
Thanks. The change is implemented locally. I'll abstain from pushing until I get answers to https://github.com/asfernandes/cloop/issues/8#issuecomment-1015381749 though.